kolibri/plugins/device/assets/src/views/FacilitiesPage/index.vue

```diff
@@ -245,6 +260,18 @@
       };
     },
     computed: {
+      options() {
+        return [
+          {
+            label: 'Import facility',
+            value: 'import_facility',
+          },
+          {
+            label: 'Create new facility',
+            value: 'create_new_facility',
+          },
+        ];
+      },
```

Comment:
It would also be good to add the `value` fields as constants in `kolibri/plugins/device/assets/src/constants.js` as they seem specific to the device plugin

Reply:
Yes, the label values should be translated strings.